TESTS/mbed_drivers/lp_timer/main.cpp

```diff
@@ -56,8 +56,9 @@ extern uint32_t SystemCoreClock;
 #define US_PER_MSEC      1000
 #define TOLERANCE_FACTOR 80000.0f
 #define US_FACTOR        1000000.0f
+#define CLOCK_MAX        130000000
```

Comment:
I think a comment based on the PR description could be added here for clarification.

Reply:
how was this value chosen, why is it sufficient (based on the description, this allows to set the minimum tolerance value for targets with clock freq higher that this value)